### PR TITLE
User account creation email change

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# Motivation and Context

# What has changed

# How to test?

# Links

### CODE REVIEW

File: _infra/helm/party/templates/deployment.yaml [hunk 1]
```diff
@@ -184,6 +184,10 @@ spec:
             value: "{{ .Values.email.templates.notifyEmailVerificationTemplate }}"
           - name: NOTIFY_REQUEST_PASSWORD_CHANGE_TEMPLATE
             value: "{{ .Values.email.templates.notifyRequestPasswordChangeTemplate }}"
+          - name: NOTIFY_VERIFY_ACCOUNT_EMAIL_CHANGE
```
Comment (insacuri) [inline _infra/helm/party/templates/deployment.yaml]: These 2 variables need to have `_TEMPLATE` at the end.  Update in here and every occurrence in the PR

File: _infra/helm/party/values.yaml [hunk 2]
```diff
@@ -52,12 +52,14 @@ rollingUpdate:
   maxUnavailable: 1
 
 email:
-  enabled: false
+  enabled: true
```
Comment (insacuri) [inline _infra/helm/party/values.yaml:55]: are you sure we should be changing this?

File: config.py [hunk 3]
```diff
@@ -42,11 +42,15 @@ class Config(object):
     NOTIFY_URL = os.getenv('NOTIFY_URL', 'http://notify-gateway-service/emails/')
     NOTIFY_EMAIL_VERIFICATION_TEMPLATE = os.getenv('NOTIFY_EMAIL_VERIFICATION_TEMPLATE', 'email_verification_id')
     NOTIFY_REQUEST_PASSWORD_CHANGE_TEMPLATE = os.getenv('NOTIFY_REQUEST_PASSWORD_CHANGE_TEMPLATE',
-                                                            'request_password_change_id')
+                                                        'request_password_change_id')
     NOTIFY_CONFIRM_PASSWORD_CHANGE_TEMPLATE = os.getenv('NOTIFY_CONFIRM_PASSWORD_CHANGE_TEMPLATE',
-                                                            'confirm_password_change_id')
+                                                        'confirm_password_change_id')
+    NOTIFY_VERIFY_ACCOUNT_EMAIL_CHANGE = os.getenv('NOTIFY_VERIFY_ACCOUNT_EMAIL_CHANGE',
```
Comment (insacuri) [inline config.py]: Previous comment about `_TEMPLATE` suffix applies here too

File: config.py [hunk 4]
```diff
@@ -42,11 +42,15 @@ class Config(object):
...
+                                                   'verify_account_email_change')
+    NOTIFY_CONFIRM_ACCOUNT_EMAIL_CHANGE = os.getenv('NOTIFY_CONFIRM_ACCOUNT_EMAIL_CHANGE',
+                                                    'confirm_account_email_change')
     NOTIFY_ACCOUNT_LOCKED_TEMPLATE = os.getenv('NOTIFY_ACCOUNT_LOCKED_TEMPLATE', 'account_locked_id')
-    SEND_EMAIL_TO_GOV_NOTIFY = _is_true(os.getenv('SEND_EMAIL_TO_GOV_NOTIFY', False))
+    SEND_EMAIL_TO_GOV_NOTIFY = _is_true(os.getenv('SEND_EMAIL_TO_GOV_NOTIFY', True))
```
Comment (insacuri) [inline config.py:53]: As with the previous comment, are you sure you want to be changing this?

File: ras_party/views/account_view.py [hunk 5]
```diff
@@ -77,6 +76,12 @@ def resend_verification_email(party_uuid):
     return make_response(jsonify(response), 200)
 
 
+@account_view.route('/resend-account-email-change-expired-token/<token>', methods=['POST'])
+def resend_account_email_change_expired_token(token):
+    response = account_controller.resend_account_email_change_verification_email_expired_token(token)
+    return make_response(jsonify(response), 200)
```
Comment (insacuri) [inline ras_party/views/account_view.py:82]: This can be rewritten to `return jsonify(response)`.  The `make_response` and the explicit 200 isn't needed

**Review by insacuri — CHANGES_REQUESTED** [review]

File: _infra/helm/party/values.yaml (hunk 2 above)
Comment (creativeappdevamit) [reply _infra/helm/party/values.yaml:55]: yes - wanted to make sure dev cluster actually sends the test email for testing and mimic the actual flow :-)

File: config.py (hunk 4 above)
Comment (creativeappdevamit) [reply config.py:53]: yes - wanted to make sure dev cluster actually sends the test email for testing and mimic the actual flow :-)

File: _infra/helm/party/templates/deployment.yaml (hunk 1 above)
Comment (creativeappdevamit) [reply _infra/helm/party/templates/deployment.yaml]: 👍

Comment (creativeappdevamit) [reply _infra/helm/party/templates/deployment.yaml]: 👍

File: _infra/helm/party/values.yaml (hunk 2 above)
Comment (akssinhaamit) [reply _infra/helm/party/values.yaml:55]: 👍

File: config.py (hunk 3 above)
Comment (creativeappdevamit) [reply config.py]: 👍

File: ras_party/views/account_view.py (hunk 5 above)
Comment (creativeappdevamit) [reply ras_party/views/account_view.py:82]: followed the same pattern used as others

**Review by insacuri — APPROVED** [review]

**Pull request merged**